README.md

```diff
@@ -43,6 +43,9 @@ A simple library to generate and retrieve OAuth2 tokens for use with Google Clou
    end
    ```
 
+Note: in case you need to dynamically start multiple Goth you can do it just providing unique `:name` identifiers that does not need to be atoms.
+Underthehood we use Registry to register the processes under the provided names.
```

Comment:
could you move this documentation here: https://github.com/peburrows/goth/blob/master/lib/goth.ex#L31? How about something like this?

```diff
-  * `:name` - the name to register the server under.
+  * `:name` - a unique name to register the server under. It can be any term.
```